akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowGroupedWithinSpec.scala

```diff
@@ -135,6 +136,13 @@ class FlowGroupedWithinSpec extends AkkaSpec with ScriptedTest {
       TestConfig.RandomTestRange foreach (_ ⇒ runScript(script, settings)(_.groupedWithin(3, 10.minutes)))
     }
 
+    "group with small groups with backpressure" in {
+      Source(1 to 10)
+        .groupedWithin(1, 1.second)
+        .throttle(1, 110.millis, 0, ThrottleMode.Shaping)
+        .runWith(Sink.seq).futureValue should ===((1 to 10).map(List(_)))
```

Comment:
good point, I make the group time to be 1 day and then it will not interfere (it is irrelevant for this test).